src/ontime/model/model.py

```diff
@@ -42,3 +50,23 @@ def predict(self, n, **params):
         """
         pred = self.model.predict(n, **params)
         return TimeSeries.from_darts(pred)
+
+    def _set_model(self, ts):
+        size_of_ts = ts.n_components
+        if issubclass(self.model.__class__, ModelMeta):
+            # Darts Models
+            self.model = DartsForecastingModel(self.model, **self.params)
+        # This take all the sklearn regressors and pipelines
+        elif issubclass(self.model, BaseEstimator):
```

Comment:
Looks like the implementation of Classes between libraries are not the same. Must-use __class__ for darts and nothing for skForecast. I have not found a common implementation.